### PR TITLE
SASS

### LINKED ISSUE #1978: Refactor CSS to use SCSS
Instead of having one long CSS file I would like to use SCSS files to create a more maintainable style approach. 

Some pros of using SCSS rather than CSS: 
 - nesting
- splitting of files (separate files for header, panels, maps, ...)
- use of variables for colours and other static values (paddings)
- way easier to read and maintain 

We'd have to use [sass-loader](https://github.com/webpack-contrib/sass-loader) and add it in the webpack config so it compiles SCSS files to CSS. Another refactor would be to minimise the CSS when it is getting deployed. 

I can have a look at this if you want, let me know what your thoughts are!

### PR DESCRIPTION
**Pull request by ovbm**
merges #2019 into master,
closes #1978

### CODE REVIEW

File: web/webpack.config.js [hunk 1]
```diff
@@ -43,10 +47,8 @@ module.exports = Object.keys(languageNames).map(locale => ({
     new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, new RegExp(`/${locale}/`)),
     // Only include current locale + en
     new webpack.ContextReplacementPlugin(/locales/, new RegExp(`/${locale}|en/`)),
-    new MiniCssExtractPlugin({
-      filename: '[name].' + (isProduction ? '[chunkhash]' : 'dev') + '.css',
-      chunkFilename: '[name].' + (isProduction ? '[chunkhash]' : 'dev') + '.css',
-    }),
+    new MiniCssExtractPlugin(`[name].${isProduction ? '[hash]' : 'dev'}.css`),
```
Comment (corradio) [inline web/webpack.config.js]: revert this line

File: web/views/pages/index.ejs [hunk 2]
```diff
@@ -47,8 +47,8 @@
         <meta name="theme-color" content="#ffffff">
 
         <!-- CSS -->
-        <link rel="stylesheet" type="text/css" href="dist/styles.<%= stylesHash %>.css">
-        <link rel="stylesheet" type="text/css" href="dist/vendor.<%= vendorStylesHash %>.css">
+        <link rel="stylesheet" type="text/css" href="dist/<%= stylesHash %>.css">
```
Comment (corradio) [inline web/views/pages/index.ejs]: revert this file

File: web/webpack.config.js (hunk 1 above)
Comment (ovbm) [reply web/webpack.config.js]: done

File: web/views/pages/index.ejs (hunk 2 above)
Comment (ovbm) [reply web/views/pages/index.ejs]: done

File: web/src/scss/content/panel/_time-slider.scss [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    margin-right: 10px;
+    margin-left: 10px;
+    height: 12px;
+    border-radius: 4px;
+    cursor: pointer;
+    background: $light-gray;
+    box-shadow: inset 0 1px 4px 0 rgba(0, 0, 0, 0.10);
+}
+  
+/* .time-slider-input:focus::-webkit-slider-runnable-track {
```
Comment (corradio) [inline web/src/scss/content/panel/_time-slider.scss]: commented code?

Comment (ovbm) [reply web/src/scss/content/panel/_time-slider.scss]: was probably copied over from css. will remove

**Pull request merged**